tests/integration/clusterProxyConfig_test.go

```diff
@@ -26,6 +28,36 @@ func (m *IntegrationSuite) TestClusterProxyConfig() {
 	// Creating the first CPC should succeed
 	err = client.Create(ctx, validCreateObj.Namespace, validCreateObj, result, metav1.CreateOptions{})
 	m.NoError(err, "Error returned during the creation of a valid clusterProxyConfig")
+
+	// Verify the API knows about the first object before we try to create a second one.
+	// Wait 2 minutes for the object to be created (should be more than ample time)
+	timeout := int64(5 * time.Minute)
+	listOptions := metav1.ListOptions{
+		Watch:          true,
+		TimeoutSeconds: &timeout,
+	}
+	watcher, err := client.Watch(ctx, validCreateObj.Namespace, listOptions)
+	m.NoError(err, "Error returned trying to watch the clusterProxyConfig")
+	if watcher != nil {
+		// If `m.NoError` fails, code still keeps running and we have a nil watcher
+		defer watcher.Stop()
+		for {
+			receivedEvent, ok := <-watcher.ResultChan()
+			if !ok {
+				logrus.Infof("Got a non-ok event from the watcher; breaking")
```

Comment:
Should a non-ok event here fail the test?

Reply:
Yes, and I figured out what to do.